scripts/base_setup.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+auto_install_message = """
+Can not find cpptraj header and libcpptraj files.
+We're trying to dowload and build libcpptraj for you, would take about 5-10 minutes.
+You can check ./cpptraj/ folder after installation.
+
+To avoid auto-installation
+--------------------------
+You should set CPPTRAJHOME or installing ./cpptraj/ in this current folder.
+
+If you want to manually install `libcpptraj`, you can download cpptraj
+development version from here: https://github.com/Amber-MD/cpptraj
+
+$ git clone https://github.com/Amber-MD/cpptraj/
+$ cd cpptraj
+$ export CPPTRAJHOME=`pwd`
+$ ./configure -shared gnu
+$ make libcpptraj
+
+and then go back to pytraj folder:
+python setup.py install
+"""
+
+def remind_export_LD_LIBRARY_PATH(build_tag, libdir):
+    if build_tag:
+        from scripts.acsii_art import batman
+        print("")
+        print("")
+        print(batman)
+        libdir = os.path.abspath(libdir)
+        print('make sure to add %s to your LD_LIBRARY_PATH \n\n'
+              'example: export LD_LIBRARY_PATH=%s:$LD_LIBRARY_PATH\n\n'
+              'try simple test: python ./runtests.py simple\n\n' %
+              (libdir, libdir))
```

Comment:
In AmberTools, this is done automatically by the `amber.sh` and `amber.csh` files, so this warning should not be printed in that case.  It will just serve to confuse.  When this is part of an Amber install, can you change this message to something along the lines of "remember to source one of these files"?